src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.FileWriterService;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static final FileWriterService fileWriter = new FileWriterServiceImpl();
+    private static final String CORRECT_PATH = "src/test/resources/report.csv";
+    private static final String WRONG_PATH = "src/src/resources/report.csv";
+    private static final String REPORT = "Hello, mentor :)";
+
+    @Test
+    public void writeToFile_successfulWrite_ok() {
+        // changed the return type from void to boolean
+        // in writeToFile method for simplicity. its OK?
+        // or is this a gross violation of SOLID principles?
```

Comment:
it would be better not to change your logic after approve but leave it, don't write questions in your code, remove comments, ask questions in specific chats